### PR TITLE
#517 Apply identity in composer

### PR DESCRIPTION
**Pull request by dab246**
https://user-images.githubusercontent.com/80730648/165474952-4986abec-1ef2-40e9-bbdc-a452f40f62b8.mov

### CODE REVIEW

File: lib/features/composer/presentation/composer_controller.dart [hunk 1]
```diff
@@ -173,6 +180,11 @@ class ComposerController extends BaseController {
           _uploadAttachmentsSuccess(success);
         } else if (success is GetEmailContentSuccess) {
           _getEmailContentSuccess(success);
+        } if (success is GetAllIdentitiesSuccess) {
```
Comment (hoangdat) [inline lib/features/composer/presentation/composer_controller.dart:183]: How about if we can not `get all identities success`? I propose we can use user as we do before.

Comment (dab246) [reply lib/features/composer/presentation/composer_controller.dart:183]: It still works the way you want it to.

**Comment by hoangdat** [comment]
IMO, you forgot to add `Bcc` of Identity when you generate email in composer

image

**Comment by dab246** [comment]
> I did not see the signature is add to the composer when changing the identity. please share the demo for this also

It is not handled in this ticket. Please note that the ticket is referred to

**Comment by dab246** [comment]
> IMO, you forgot to add `Bcc` of Identity when you generate email in composer
> 
> image
> 
> Refs: https://jmap.io/spec-mail.html#identities

Please see it in more detail inside jmap_dart_client

**Comment by dab246** [comment]
> > > I did not see the signature is add to the composer when changing the identity. please share the demo for this also
> > 
> > 
> > It is not handled in this ticket. Please note that the ticket is referred to
> 
> srr, please do it later in: #457

**Review by hoangdat — APPROVED** [review]

**Pull request merged**